src/schema/v2/me/artistRecommendations.ts

```diff
@@ -0,0 +1,84 @@
+import { GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { connectionFromArraySlice } from "graphql-relay"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { CursorPageable, pageable } from "relay-cursor-paging"
+import { createPageCursors } from "schema/v1/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import gql from "lib/gql"
+import { artistConnection } from "schema/v2/artist"
+import { find } from "lodash"
+
+const MAX_ARTISTS = 50
+
+export const ArtistRecommendations: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  description: "A connection of artist recommendations for the current user.",
+  type: artistConnection.connectionType,
+  args: pageable({
+    page: { type: GraphQLInt },
+  }),
+  resolve: async (
+    _root,
+    args: CursorPageable,
+    { vortexGraphqlLoader, artistsLoader }
+  ) => {
+    if (!vortexGraphqlLoader || !artistsLoader) return
+
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
+    // Fetch artist IDs from Vortex
+
+    const vortexResult = await vortexGraphqlLoader({
+      query: gql`
+        query artistRecommendationsQuery {
+          artistRecommendations(first: ${MAX_ARTISTS}) {
+            totalCount
+            edges {
+              node {
+                artistId
+                score
+              }
+            }
+          }
+        }
+      `,
+    })()
+
+    const artistIds: string[] = vortexResult.data?.artistRecommendations?.edges?.map(
+      (edge) => edge?.node?.artistId
+    )
```

Comment:
**nitpick:** I saw this pattern being used a few times in metaphysics, maybe it's time to add `extractNodes` as a helper like we do in Eigen https://github.com/artsy/eigen/blob/5b6d470e1177ced6dbc997cc3401d9876cfcbed0/src/lib/utils/extractNodes.ts

Reply:
Good idea! I added an `extractNodes` helper 👍